### PR TITLE
add -q to bypass prompts

### PR DESCRIPTION
**Pull request by furkanpehli1**
Added `--auto` argument to run cmd args automatically.

Waiting for your comments :)

### CODE REVIEW

**Comment by ahmetb** [comment]
Doesn't make sense to me. Empty regex already runs on all + what's automatic here.

**Comment by furkanpehli1** [comment]
It is just for bypassing `Continue? [Y/n]:` question like `--auto-approve` in terraform apply/destroy.

**Comment by ahmetb** [comment]
There's an env var for that. It's not documented yet. You can also bypass that by piping "yes|" to it. 

I'd be open to -q for bypassing the prompt.

**Comment by furkanpehli1** [comment]
Yeah I'm aware there is env var but that would be pretty if there  be option like `-q` like you mentioned. If you are open to it I can make some changes on my PR.

**Comment by furkanpehli1** [comment]
Due to feedback, changes have made. I'm extremely open to new reviews :)

**Comment by ahmetb** [comment]
Looks good. You can reopen if you want.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ kubectl allctx -c 2 '~' -- version
 
 # specify where the context name gets passed in the command executed (_ replaced with context name)
 kubectl allctx -I _ '~test$' -- my_plugin -ctx=_
+
+# Bypass prompt with using '-q' flag
```
Comment (ahmetb) [inline README.md:22]: You can remove this from here. -h shows it anyway

File: main.go [hunk 2]
```diff
@@ -44,10 +44,11 @@ var (
 
 	repl    = flag.String("I", "", "string to replace in cmd args with context name (like xargs -I)")
 	workers = flag.Int("c", 0, "parallel runs (default: as many as matched contexts)")
+	quiet   = flag.Bool("q", false, "bypasses prompt")
```
Comment (ahmetb) [inline main.go:47]: accept confirmation prompts

**Review by ahmetb — CHANGES_REQUESTED** [review]
Overall looks good. Cosmetics feedback.

File: main.go [hunk 3]
```diff
@@ -44,10 +44,11 @@ var (
...
 )
 
 func logErr(msg string) {
-	fmt.Fprintf(os.Stderr, red("error: "))
+	fmt.Fprintf(os.Stderr, "%s", red("error: "))
```
[the remark this answers is not in the document]
Comment (furkanpehli1) [reply main.go:51]: I was getting following warning from go-staticcheck on my ide;

`printf-style function with dynamic format string and no further arguments should use print-style function instead (SA1006)`

So I applied what [documentation](https://staticcheck.io/docs/checks#SA1006) says. After all it's just a linter warning and not a big deal. If you want I can revert it to the original.

[the remark this answers is not in the document]
Comment (ahmetb) [reply main.go:51]: I merged this but realized you don't even use the `msg` here. Consider installing a linter to your editor.

**Pull request merged**